### ISSUE TITLE
More than 2 parents

### ISSUE THREAD

**Issue by terrorist96** [liberland/constitution#478]
> §V.1. Any Person who has fulfilled relevant requirements, as prescribed by law, shall be eligible to obtain citizenship of the Free Republic of Liberland; **should a Person be born to parents of whom both are Citizen of the Free Republic of Liberland, he or she shall automatically be entitled to citizenship thereof**; multiple citizenship shall be allowed; no Citizen shall be deprived of his or her citizenship unless by the Criminal Court upon conviction for a criminal offence and as a part of punishment as prescribed by law; no Person shall be rendered stateless in any event.

There is a problem. In the future, a child may have more than two biological parents. It's already happening.
http://www.bbc.com/news/magazine-28986843

How can we reword it to say `all parents` instead of `both parents`?

**Comment by KacperZajc** [comment]
As far as I know the 3rd parent gives only less than 1% of DNA.
There is no way he/she can obtain any custody rights.
But no harm done changing it, although it will sound odd.

**Comment by liberlandcitizen** [comment]
Is there a reason we are requiring two parents rather than just one to have
citizenship in order for the child to gain citizenship?

**Comment by KacperZajc** [comment]
Make it into a PR

**Comment by terrorist96** [comment]
> Make it into a PR

Done.

> Is there a reason we are requiring two parents rather than just one to have
citizenship in order for the child to gain citizenship?

Yes, because the child may be able to obtain the citizenship of the non-liberland-citizen parent.

**Comment by liberlandcitizen** [comment]
<<Yes, because the child may be able to obtain the citizenship of the
non-liberland-citizen parent.>>

But why do we want a child to have the citizenship of the other parent's
country and not Liberland?

**Comment by terrorist96** [comment]
Because we covet our citizenship. Kacper can expound upon that if he wants.

**Comment by liberlandcitizen** [comment]
As well we should covet our citizenship. However, I find it strange that we
would get into the business of who our citizens choose to procreate with
and give extra benefit to those who happen to sleep with a fellow citizen
versus those who create life with somebody who is not a citizen.

**Comment by terrorist96** [comment]
A baby with 1 citizen parent can still obtain citizenship; it just won't be automatic. Say you have one Liberland citizen and one USA citizen. USA deems the child its citizen since one of its parents is a citizen. If we give automatic citizenship, then the child would have 2 citizenships by default.

**Comment by liberlandcitizen** [comment]
There's nothing wrong with having two citizenships by default. Using myself
as an example, if I had kids they would have US and Italian citizenship by
default without any contradictions or issues.

The Liberland constitution clearly allows dual citizenship, so what's the
point/problem here?

**Comment by terrorist96** [comment]
I don't know, but this is how it was decided. @KacperZajc can you explain the reasoning?

**Comment by KacperZajc** [comment]
We want to preserve as much as possible the power to give citizenship so that we can distribute it on merits rather than by the automatic entitlement. 
In any event the ultimate decision on the wording of this provision rests with the President.
For now it stays as it is.

**Comment by liberlandcitizen** [comment]
I suggest it be given to children with one or more Liberland parent, or to
no children at all. That way it's fair. There should be no added benefit of
procreating with a citizen versus a non-citizen. I hope the President will
consider this point.

**Comment by KacperZajc** [comment]
The problem with the former is that if the parents give up their original citizenship and be only citizens of the FRL, then the child will be stateless on birth until it can somehow earn our citizenship. This is unacceptable. Hence the current version which is allows from a higher degree of control over the distribution of citizenship yet prevent any child being born stateless.

Of course you can always make an argument that parents can buy it for the child or not give up their original citizenship, but statelessness has such serious consequences  that it should not be left up to the good will of the parents.

**Comment by liberlandcitizen** [comment]
In that case we could make an exception if it renders the child stateless,
as we have made elsewhere in the constitution.

**Comment by KacperZajc** [comment]
your opinion will be taken into consideration
